pkg/kubelet/nodeshutdown/nodeshutdown_manager_linux.go

```diff
@@ -287,15 +287,18 @@ func (m *managerImpl) start() (chan struct{}, error) {
 
 					m.processShutdownEvent()
 				} else {
-					m.aquireInhibitLock()
+					if err := m.acquireInhibitLock(); err != nil {
+						m.logger.V(1).Error(err, "Failed to acquire inhibit lock")
+						return
```

Comment:
This changes the behavior, more than just a typo fix.  Can you update the commit message about why this is correct?

Reply:
Fair. I'd rather not change the existing behavior. I'll update and discard the return value to keep the linter happy.